src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -706,6 +702,10 @@ public function link($method = 'get')
 
         $node = $this->getNode(0);
 
+        if (!$node instanceof \DOMElement) {
+            throw new \InvalidArgumentException(sprintf("The current node list should contain only DOMElement instances, '%s' found.", get_class($node)));
```

Comment:
For consistency, we should use single quotes to enclose the message, and double quotes inside the message: 

```php
throw new \InvalidArgumentException(sprintf('The current node list should contain only DOMElement instances, "%s" found.', get_class($node)));
```